src/clr-angular/forms/select/options.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component, ElementRef, HostBinding, Inject, Injector, OnDestroy, Optional} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {AbstractPopover} from "../../popover/common/abstract-popover";
+import {Point} from "../../popover/common/popover";
+import {POPOVER_HOST_ANCHOR} from "../../popover/common/popover-host-anchor.token";
+import {IfOpenService} from "../../utils/conditional/if-open.service";
+
+@Component({selector: "clr-options", templateUrl: "options.html", host: {"[class.clr-options]": "true"}})
+export class ClrOptions extends AbstractPopover implements OnDestroy {
+    // Have to declare this separately or TS complains that super call in the constructor
+    // should be before the if condition which we do not want in this case
+    protected ifOpenService: IfOpenService;
+    private sub: Subscription;
+
+    constructor(injector: Injector, @Optional() @Inject(POPOVER_HOST_ANCHOR) parentHost: ElementRef,
+                ifOpenService: IfOpenService) {
+        if (!parentHost) {
+            throw new Error("clr-select-menu should only be used inside of a clr-select");
+        }
+        super(injector, parentHost);
+
+        // Initialize Services
+        this.ifOpenService = ifOpenService;
+
+        // Configure Popover
+        this.initializeSubscriptions();
+        this.configurePopover();
+    }
+
+    @HostBinding("class.open")
```

Comment:
The other popover components all have the `.open` class on the parent component, not the popover itself. Any reason why you changed this for select? I'm okay either way, I'm mostly curious.

Reply:
This was a mistake but now that I think about it, if I move it to the parent it creates a nesting in the CSS. Even though its just 2 levels of nesting, I feel that its unnecessary.

So instead of:
```
.clr-select.open .clr-options {
    //do whatever
}
```

I can just do:
```
.clr-options.open {
    //do whatever
}
```

So i think we should keep this pattern instead of what we normally do. Do you think thats alright?

Reply:
Do we style the input in any way when the menu is open? Or do we plan on doing so?

I'm not even sure what you're going to use the selector you just showed for. I don't see any styles that apply to the menu only when it is open, since... well... when it's not open it's not displayed.